app/agents/voice/automatic/__init__.py

```
@@ -115,12 +115,28 @@ async def main():
+    if config.ENABLE_AIC_FILTER and config.AICOUSTICS_LICENSE_KEY:
+        try:
+            aic_filter = AICFilter(
+                license_key=config.AICOUSTICS_LICENSE_KEY,
+                enhancement_level=config.AIC_ENHANCEMENT_LEVEL,
+                voice_gain=config.AIC_VOICE_GAIN,
+                noise_gate_enable=config.AIC_NOISE_GATE_ENABLE,
+            )
+            daily_params.audio_in_filter = aic_filter
+            logger.info(f"AIC Filter: ENABLED (enhancement_level={config.AIC_ENHANCEMENT_LEVEL}, voice_gain={config.AIC_VOICE_GAIN}, noise_gate={config.AIC_NOISE_GATE_ENABLE})")
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)